lua/compe_emoji/gen-data.py

```diff
@@ -0,0 +1,32 @@
+# Generates the emoji data LUA file using this as a source:
+# https://github.com/iamcal/emoji-data/blob/master/emoji.json.
+import json
+
+
+def write_item(char, name, output):
+    output.write("table.insert(items, {")
```

Comment:
Why not construct and return the table directly instead? E.g. in `write_item`:

```python
output.write(f"  {{word = '{char}', abbr = '{char} :{name}:', filter_text = ':{name}:'}},\n")
```
In `write_data`:

```python
output.write("return {\n")
# for loop with write_item
output.write("}\n")
```

This way, you don't do `table.insert` for each and every emoji.

Reply:
That's actually quite smart. For some reason I though it might be problematic to have such a huge statement, but it's fine. Thanks!